pkgs/test/lib/src/executable.dart

```diff
@@ -1,179 +1,19 @@
-// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Why remove the copyright?

Reply:
This file was moved and I started one from scratch. I added the copyright back.